### DISCUSSION TITLE
Port Forwarding two machines

### DISCUSSION THREAD

**Discussion by airededesdeux** [isontheline/pro.webssh.net#464] [Q&A] [answered]
Hello Hello, 

I'm prefer to warm,I'm a begginer, sorry in advance if my question is stup*d.

This is my problem :

I have a VPS on internet :  5.X.X.X
On this VPS I have an openvpn server under the network (10.8.0.0/24)

I would like to access to a new subnet (192.168.18.0/24). 

And If I'm correct I need to go 
   => VPS => Raspberry => Target Subnet

I think a picture is more helper :) : 

![IMG_0009](https://user-images.githubusercontent.com/69821379/144301710-b79e21c8-b55d-4f5d-930d-755dd2a545b8.jpg)

And to be honest I don't understand if webssh can help me with this problem at this time. 

My target subnet cannot store any webservice or any VPN service.

Thanks a lot for your time and your advice ;) 

++

**Comment by isontheline** [comment, marked as answer]
Bonjour @airededesdeux 👋

Actuellement la fonctionnalité "JumpHost" #266 n'est malheureusement pas encore développée. 
Cette fonctionnalité est nécessaire car dans votre exemple il faut effectuer plusieurs rebonds et WebSSH ne le permet pour le moment pas.

Elle devrait voir le jour dans une prochaine version, je ne peux néanmoins pas indiquer de date de disponibilité.

**Comment by airededesdeux** [comment]
Bonjour @isontheline Merci de m'avoir répondu. Je comprends tout à fait, je vais patienter dans ce cas. :) 

En l'état peut-on imaginer l'équivalent de la commande : 

ssh -p 2222 -L 80:192.168.1.80:80 www.example.com 

Sur WebSSH s'il vous plaît?

Merci :)

**Reply by isontheline** [reply]
La documentation sur le port forwarding est disponible ici :
https://webssh.net/documentation/help/networking/port-forwarding/

Est-ce que cette page répond à votre besoin en terme de redirection de port ?

**Reply by airededesdeux** [reply]
Parfait, elle répond à ma question ;) 
Merci à vous!
